server/src/main/java/org/elasticsearch/index/query/CoordinatorRewriteContextProvider.java

```diff
@@ -16,29 +16,29 @@
 import org.elasticsearch.index.Index;
 import org.elasticsearch.index.mapper.DateFieldMapper;
 import org.elasticsearch.index.shard.IndexLongFieldRange;
-import org.elasticsearch.xcontent.NamedXContentRegistry;
+import org.elasticsearch.xcontent.XContentParserConfiguration;
```

Comment:
Oh yikes this snuck in here! That was not my intent....

Reply:
It's supposed to be in https://github.com/elastic/elasticsearch/pull/80161 - which was approved. So I'm not going to revert.

Reply:
But I won't backport it.